src/img/feedback.svg

```diff
@@ -2,5 +2,5 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <svg data-name="Flat (For Export)" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
-  <path fill="context-fill" d="M15.93 14.24l-2-3.37A5.75 5.75 0 0 0 15 7.5C15 3.91 11.64 1 7.5 1S0 3.91 0 7.5 3.36 14 7.5 14a8.47 8.47 0 0 0 3.13-.6l4.71 1.6a.45.45 0 0 0 .16 0 .51.51 0 0 0 .39-.19.5.5 0 0 0 .04-.57zM3.5 9A1.5 1.5 0 1 1 5 7.5 1.5 1.5 0 0 1 3.5 9zm4 0A1.5 1.5 0 1 1 9 7.5 1.5 1.5 0 0 1 7.5 9zm4 0A1.5 1.5 0 1 1 13 7.5 1.5 1.5 0 0 1 11.5 9z"/>
+  <path fill="context-fill" fill-opacity="context-fill-opacity" d="M15.93 14.24l-2-3.37A5.75 5.75 0 0 0 15 7.5C15 3.91 11.64 1 7.5 1S0 3.91 0 7.5 3.36 14 7.5 14a8.47 8.47 0 0 0 3.13-.6l4.71 1.6a.45.45 0 0 0 .16 0 .51.51 0 0 0 .39-.19.5.5 0 0 0 .04-.57zM3.5 9A1.5 1.5 0 1 1 5 7.5 1.5 1.5 0 0 1 3.5 9zm4 0A1.5 1.5 0 1 1 9 7.5 1.5 1.5 0 0 1 7.5 9zm4 0A1.5 1.5 0 1 1 13 7.5 1.5 1.5 0 0 1 11.5 9z"/>
```

Comment:
While the `fill` style has an effect on this icon, I don't see an effect from the `fill-opacity` style. I guess this makes sense, since we only apply a `fill`, and not an `opacity` here:

```css
.button.ghost .icon {
  -moz-context-properties: fill;
  fill: var(--grey-90-a80);
  margin: 0;
}
```

In otherwords, I don't think we need to add `fill-opacity="context-fill-opacity"` to the ghost button SVGs inside the popup, since we're not specifying an `opacity` style, but if we want to do that say for future-proofing, then we would also need to add `fill-opacity` to our list of `-moz-context-properties`:

```css
.button.ghost .icon {
  -moz-context-properties: fill, fill-opacity;
  fill: var(--grey-90-a80);
  margin: 0;
}
```